pages/apis/webhooks/agent_events.md

```diff
@@ -32,3 +32,33 @@ Example request body:
   }
 }
 ```
+
+### `agent.blocked`
```

Comment:
This is a different section because it has different fields to the rest of the agent events

Reply:
This doesn't sit right with me in the hierarchy, but my Friday-afternoon brain can't solve it. I'll need to come back to this on Monday. Maybe it just needs to be an H2? Maybe there needs to be a new heading that groups the other events as something separate from this one?